spec/models/service_template_ansible_playbook_spec.rb

```diff
@@ -1,8 +1,8 @@
 describe ServiceTemplateAnsiblePlaybook do
   let(:user)     { FactoryGirl.create(:user_with_group) }
-  let(:auth_one) { FactoryGirl.create(:authentication, :manager_ref => 6) }
-  let(:auth_two) { FactoryGirl.create(:authentication, :manager_ref => 10) }
-  let(:auth_three) { FactoryGirl.create(:authentication, :manager_ref => 14) }
+  let(:auth_one) { FactoryGirl.create(:embedded_ansible_credential, :manager_ref => 6) }
+  let(:auth_two) { FactoryGirl.create(:embedded_ansible_credential, :manager_ref => 10) }
+  let(:auth_three) { FactoryGirl.create(:embedded_ansible_credential, :manager_ref => 14) }
```

Comment:
Same here, `manager_ref` always be set to a string.  Then `native_ref` converts to the expected type based on the sub-class.

Reply:
We should actually update the parser to convert to match the Db column then.  Standard graph refresh will do the casting but batch graph refresh will not